requirements/common.in

```diff
@@ -155,7 +155,7 @@ python-twitter==3.3
 polib==1.1.0
 
 # Needed for cloning virtual environments
--e git+https://github.com/zulip/virtualenv-clone.git@44e831da39ffb6b9bb5c7d103d98babccdca0456#egg=virtualenv-clone==0.2.6.zulip1
+-e git+https://github.com/zulip/virtualenv-clone.git@32eafad8979eb379458ec118d27ad451599db451#egg=virtualenv-clone==0.3.0.zulip1
```

Comment:
We should update these to just use the release from pypi, not the tag from GitHub.

Generally, using a release is better (e.g. `pip` does much better caching)